content/blog/2024-06-27-using-bitnami-valkey-chart/index.md

```diff
@@ -0,0 +1,182 @@
++++
+title="Using Bitnami's Valkey chart"
+date=2024-06-27 01:01:01
+description="An introductory example using Bitnami's Valkey chart"
+aliases= [
+    "/blog/2024/06/using-bitnami-valkey-chart/"
+]
+authors=["rafariossaa"]
++++
+
+Valkey is a high-performance key/value datastore that supports workloads such as caching, and message queues, supporting many data types including strings, numbers, hashes, bitmaps, and more. Valkey can run in standalone or cluster mode for replication and high availability.
+
+This blog will serve as walk-through on how you can deploy and use the [Bitnami Helm chart for Valkey](https://github.com/bitnami/charts/tree/main/bitnami/valkey).
```

Comment:
Can you write a sentence or two about _why_ someone would want to use the Bitnami helm chart?